docs/plugins/contributors/users.json

```diff
@@ -10,6 +10,7 @@
     "github": "https://github.com/CollierCZ",
     "twitter": "",
     "website": "https://collier.cz/",
+    "core": true,
```

Comment:
Is this property new? Why do we need it? 🤔

Reply:
yeah, to distinguish core team from other contributors :)